gateway/api/migrations/0015_program_public.py

```diff
@@ -0,0 +1,18 @@
+# Generated by Django 4.2.2 on 2024-01-21 14:30
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ("api", "0014_runtimejob"),
+    ]
+
+    operations = [
+        migrations.AddField(
+            model_name="program",
+            name="public",
+            field=models.BooleanField(default=False),
+        ),
```

Comment:
Assuming we may add other groups in the future, should we maybe make this a bit more generic? i.e. instead of a boolean using a text field named `group`, which we could set to `public`

Reply:
I thought about it.  I think that we can use the `public` as the master switch for the sharing.  The `group` can be added later for fine access control.